azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/telemetry/GetHashMac.java

```diff
@@ -85,6 +88,29 @@ private static String getRawMac() {
         return ret;
     }
 
+    private static String getRawMacWithoutIfconfig() {
+        final List<String> macSet = new ArrayList<>();
+        try {
+            final Enumeration<NetworkInterface> interfaces = NetworkInterface.getNetworkInterfaces();
```

Comment:
log an event when ifconfig is not available(this code is called)

Reply:
Done

Reply:
This method is called while Telemetry Proxy is initializing, so we can't send telemetries here.